code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -22,6 +23,69 @@ namespace PlayFab
 {
     namespace QoS
     {
+        bool ValidateResult(PlayFabResultCommon& resultCommon, CallRequestContainer& container)
```

Comment:
Can the container be const&?

Reply:
No, the method call below "FromJson" expect a non-const&